src/EventTeachingService.cpp

```diff
@@ -11,259 +11,427 @@
 
 namespace VLCB {
 
-void EventTeachingService::setController(Controller *cntrl)
+void EventTeachingService::setController(Controller *cntrl) 
 {
   this->controller = cntrl;
   this->module_config = cntrl->module_config;
 }
 
-void EventTeachingService::enableLearn()
+void EventTeachingService::enableLearn() 
 {
   bLearn = true;
-   //DEBUG_SERIAL << F("> set learn mode ok") << endl;
+  DEBUG_SERIAL << F("> set learn mode ok") << endl;
```

Comment:
Please comment out the debug outputs again once you are done with trouble shooting.

Reply:
Sorry, forgot that I'd uncommented some debugs!